src/payment/strategies/humm/humm-payment-strategy.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { createClient as createPaymentClient } from '@bigcommerce/bigpay-client';
+import { createAction, createErrorAction, Action } from '@bigcommerce/data-store';
+import { createFormPoster, FormPoster } from '@bigcommerce/form-poster';
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { createScriptLoader } from '@bigcommerce/script-loader';
+import { merge } from 'lodash';
+import { of, Observable } from 'rxjs';
+
+import { createCheckoutStore, createInternalCheckoutSelectors, CheckoutRequestSender, CheckoutStore, CheckoutValidator } from '../../../checkout';
```

Comment:
after applying all the recommendations to the test cases please be aware of fixing any imports that need to be reorganized